### DISCUSSION TITLE
Dark mode

### DISCUSSION THREAD

**Discussion by zMotoR** [ghiscoding/Angular-Slickgrid#1326] [Ideas]
Hello. Do you plan to support Bootstrap dark mode that was appeared with v5.3.0? - https://getbootstrap.com/docs/5.3/customize/color-modes/#dark-mode

**Comment by ghiscoding** [comment]
This is a free Open Source project, if you want something then please Contribute to the project. The short answer is No unless I have contributions for it. There are plenty of [SASS variables](https://github.com/ghiscoding/slickgrid-universal/blob/master/packages/common/src/styles/_variables.scss), it shouldn't be too hard to create a Dark Theme from it
